src/test/java/edu/ucsb/cs156/example/controllers/CollegiateSubredditsTests.java

```diff
@@ -0,0 +1,5 @@
+package edu.ucsb.cs156.example.controllers;
+
+public class CollegiateSubredditsTests {
+    
+}
```

Comment:
This file should be removed. We already have a `CollegiateSubredditsControllerTests.java`
```suggestion
```